wgsl/index.bs

```diff
@@ -9879,10 +9879,18 @@ For each [=formal parameter=] of a function, one or two tags are computed:
   <thead>
     <tr><th>Parameter Tag<th>Description
   </thead>
-  <tr algorithm="ParameterRequiredToBeUniform tag">
+  <tr algorithm="ParameterRequiredToBeUniformValue tag">
       <td><dfn noexport>ParameterRequiredToBeUniform.S</dfn>,<br>
           where |S| is one of the severities: [=severity/error=], [=severity/warning=], or [=severity/info=].
       <td>The parameter must be a [=uniform value=].
+          If the parameter type is a pointer, the memory view must be uniform.
```

Comment:
Can we indicate somehow that we're not making a judgement about the contents of the referenced locations?

> If the parameter is a pointer, the memory view, but not necessarily its contents, must be uniform.